ios/BT/ExposureManager.swift

```diff
@@ -525,4 +523,61 @@ private extension ExposureManager {
     localUncompressedURLs = []
     downloadedPackages = []
   }
+
+  func processExposureDetectionSummary(_ summary: ENExposureDetectionSummary?, completion: @escaping (([ExposureDetectionSummary]) -> Void)) {
+    if let summary = summary {
+
+      // Serialize exposure detection summary
+      let exposureDetectionSummary = ExposureDetectionSummary(summary)
+
+      // Prune exposure detection summaries >14 days from realm
+      BTSecureStorage.shared.pruneExposureDetectionSummaries()
```

Comment:
can we use btSecureStorage here to make this testable?

Reply:
sorry - yes, this was from a commit before your refactor 👍